docs/source/common_issues.rst

```diff
@@ -382,6 +382,22 @@ More specifically, mypy will understand the use of ``sys.version_info`` and
    else:
        # Other systems
 
+As a special case, you can also use one of these checks in a top-level
+``assert``; this makes mypy skip the rest of the file.  Example:
```

Comment:
Maybe explicitly mention that the assert must be unconditional and not nested within a statement.